Assignment-Day-6/Question2.js

```diff
@@ -0,0 +1,40 @@
+// Create a class Shape with properties width and height and methods getArea(). Create two classes Rectangle and Triangle that inherit from the Shape class and implement the getArea() method for their respective shapes.
+
+class Shape {
+  constructor(width, height) {
+    this.width = width
+    this.height = height
+  }
+
+  getArea() {
+    return NaN
+  }
```

Comment:
why return NaN?

Reply:
I had added NaN as we can return the area only when we what type of shape the calling object.